types/react-dates/README.md

```diff
@@ -0,0 +1,9 @@
+# @types/react-dates
+
+## Adding a new entry
+
+When a new component or util has been published by `react-dates`:
+
+- Add the type to `index.d.ts`
+- Add your entry import/re-export in `esm` & `lib` folders
+- Add `esm/<type>.d.ts` & `src/<type>.d.ts` to `OTHER_FILES.txt`
```

Comment:
This should this say lib/<type>.d.ts, not src/<type>.d.ts right?

Reply:
Aside from this, looks good. Thanks for contributing!

Reply:
Yes. I've changed it to `lib/<type>.d.ts`